catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -512,3 +527,10 @@ def _commit_records(self) -> int:
             total += store.commit()
         logger.info(f"Committed {total} records")
         return total
+
+    def _verbose_log(self, msg: str, data=[]):
+        if self._should_verbose_log:
+            logger.info(msg)
+            if data:
+                for item in data[:5]:
+                    logger.info(f"\t{item}")
```

Comment:
I love the added formatting, much visually cleaner! However instead of logging each line separately, I think we should do this all in one log statement by instead appending the items from `data` to the `msg` before logging. Something like:
```
if data:
    msg += "\n".join([f"\t{item}" for item in data])
```

Reply:
I pushed code that does all this in one log statement now